services/apis/drs/drsProps.js

```diff
@@ -28,11 +28,13 @@ module.exports = {
     status: 404,
   }),
 
-  noAccessToken: new Gen3Response({
+  noAccessTokenOld: new Gen3Response({
     request: {},
-    body: { msg: 'Not Authorized. Access Token Required.' },
-    data: { msg: 'Not Authorized. Access Token Required.' },
     status: 403,
   }),
 
+  noAccessToken: new Gen3Response({
+    request: {},
+    status: 401,
```

Comment:
while running with fence `master`, the `@drsNew` test failed with:
```
expected
{ fenceError: undefined,   parsedFenceError: undefined,   body:
     { msg: 'Not Authorized. Access Token Required.',      status_code: 403 },
   data: 
    { msg: 'Not Authorized. Access Token Required.',
      status_code: 403
 },   status: 403,
   requestMethod: 'GET',
   requestHeaders: undefined,
   requestBody: undefined,
   requestURL: undefined
 }
to have property 'status' of 401, but got 403
```

Reply:
I think that's b/c this isn't merged https://github.com/uc-cdis/cloud-automation/pull/1726 it's still using the old routing logic